src/lib/flags.ts

```diff
@@ -4,4 +4,5 @@ type FeatureFlags = {
 
 export const featureFlags: FeatureFlags = {
   gigs: false,
+  sessionReplay: process.env.NODE_ENV === 'development' ? false : true,
```

Comment:
```suggestion
  sessionReplay: process.env.NODE_ENV === 'production',
```

Why not just enable it in production? And not staging?